libs/dao/src/main/java/com/akto/dao/SingleTypeInfoDao.java

```diff
@@ -306,6 +307,48 @@ public List<ApiInfo.ApiInfoKey> fetchEndpointsInCollection(int apiCollectionId)
         return endpoints;
     }
 
+    public List<ApiInfo.ApiInfoKey> fetchEndpointsInCollection2(SingleTypeInfo.SubType subType, int skip, int limit) {
```

Comment:
can we refractor this to use the existing function, with a simillar functionality.

Reply:
Done